qutebrowser/commands/command.py

```diff
@@ -33,12 +33,13 @@ class ArgInfo:
 
     """Information about an argument."""
 
-    def __init__(self, win_id=False, count=False, flag=None, hide=False,
-                 metavar=None, completion=None, choices=None):
+    def __init__(self, win_id=False, count=False, hide=False, metavar=None,
+                 zero_count=False, flag=None, completion=None, choices=None):
         if win_id and count:
             raise TypeError("Argument marked as both count/win_id!")
```

Comment:
Similar to this, we should fail if `zero_count=True` is given without `count=True` because that combination makes no sense.

Please also add a test for that scenario to `tests/unit/commands/test_cmdutils.py` (similar to `TestArgument.test_count_and_win_id_same_arg`).